modules/microshift-security-context-constraints.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * microshift_running_apps/microshift-authentication.adoc
+
+:_content-type: CONCEPT
+[id="microshift-security-context-constraints_{context}"]
+
+= Security context constraint synchronization with pod security standards
+
+{product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Globally, the `restricted` profile is enforced by default for {product-title}.
+
+In addition to the global pod security admission control configuration, a controller exists that applies pod security admission control `warn` and `audit` labels to namespaces according to the SCC permissions of the service accounts that are in a given namespace.
```

Comment:
Should the SCC acronym be spelled out here since this is the first occurrence ? I see it in the next stanza.